dir.props

```diff
@@ -27,8 +27,8 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <RoslynVersion>1.0.0-rc3-20150510-01</RoslynVersion>
-    <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
+    <RoslynVersion>2.0.0-beta3</RoslynVersion>
```

Comment:
I intended to put up an md documenting how to update the compilers, but I think your suggestion also makes sense, I'll follow up with a change to buildtools and corefx PR's.